src/acom_music_box/conditions.py

```diff
@@ -43,8 +34,8 @@ def __init__(
         """
         self.pressure = pressure
         self.temperature = temperature
-        self.species_concentrations = species_concentrations if species_concentrations is not None else []
-        self.reaction_rates = reaction_rates if reaction_rates is not None else []
+        self.species_concentrations = species_concentrations if species_concentrations is not None else {}
```

Comment:
Could you update the description in `line 32-33` to reflect the changes in the data container? 

```
species_concentrations (List[SpeciesConcentration]): A list of species concentrations. Default is an empty list.
reaction_rates (List[ReactionRate]): A list of reaction rates. Default is an empty list.
```

Reply:
Ah, yes. Thanks for catching that!